src/notifier.js

```diff
@@ -1005,6 +1012,14 @@ function _processPayload(url, accessToken, payload, callback) {
   // go ahead and send it.
   XHR.post(url, accessToken, payload, function xhrCallback(err, resp) {
     if (err) {
+      if (err instanceof ConnectionError) {
```

Comment:
We will want to check to make sure that `ConnectionError` is not `undefined` before doing this `instanceof`